examples/advanced/03_web_frameworks/streamlit_project/app.py

```
@@ -0,0 +1,503 @@
+        
+        tool_option = st.selectbox(
+            "도구 선택",
+            ["번역기", "코드 생성", "SQL 쿼리 생성", "정규식 생성"]
+        )
+        
+        if tool_option == "번역기":
+            col1, col2 = st.columns(2)
+            
+            with col1:
+                source_text = st.text_area("원문", height=200)
+                source_lang = st.selectbox("원문 언어", ["자동 감지", "한국어", "영어", "일본어", "중국어"])
```

Ruff (0.11.9)

Local variable `source_lang` is assigned to but never used

Remove assignment to unused variable `source_lang`

(F841)